src/lib/provider/RUIProvider.jsx

```diff
@@ -1,22 +1,28 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, {
+  useMemo,
+} from 'react';
 import defaultTranslations from '../translations/en';
 import RUIContext from './RUIContext';
 
 const RUIProvider = ({
   children,
   globalProps,
   translations,
-}) => (
-  <RUIContext.Provider
-    value={{
-      globalProps,
-      translations,
-    }}
-  >
-    {children}
-  </RUIContext.Provider>
-);
+}) => {
+  const childProps = useMemo(() => ({
+    globalProps,
+    translations,
+  }), [globalProps, translations]);
+  console.log(childProps);
```

Comment:
Remove console.log.

Also I am not sure about this memo. It can compare shallow objects only, so it might be useless to use useMemo here.

Reply:
> Remove console.log.

done

> Also I am not sure about this memo. It can compare shallow objects only, so it might be useless to use useMemo here.

I did some testing on the `/docs/customize/global-props/` page, and I think it works as expected. The reason is not shallow compare, but referential equality.